extensions/amp-dailymotion/0.1/test/test-amp-dailymotion.js

```diff
@@ -53,13 +53,16 @@ describes.realWin(
     });
 
     it('renders responsively', async () => {
+      // optResponsive: true
       const dailymotion = await getDailymotion('x2m8jpp', true);
```

Comment:
FYI the way that I've seen test files annotate is:
```js
      const dailymotion = await getDailymotion('x2m8jpp', /* optResponsive */ true);
```

Reply:
Ah Thanks! That is way more readable!